### DISCUSSION TITLE
023BE6B8-69F3-44D1-88DC-E14968E9A467

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#179] [General]
# 023BE6B8-69F3-44D1-88DC-E14968E9A467

Kepler-1746, also known as KOI-1375, is an F-type subgiant star about 3,287 light-years away in the constellation Lyra. A single neptunian planet is orbiting at a distance of 0.88 AU from its parent star

https://celestia.mobi/addon?item=023BE6B8-69F3-44D1-88DC-E14968E9A467

**Comment by FaithfulCocoa** [comment]
thats just jupiter...
